EMRM.py

```diff
@@ -36,14 +51,25 @@ def __init__(self, characters, wlen, input_file):
         self.pos_word_chr = self.get_onehot_chr_rep()
 
     def get_onehot_chr_rep(self):
+        """Get one-hot representation of each character at each position for each word."""
         pos_word_chr = np.zeros([self.wlen, self.vocab_size, self.nchar])       
         for w in self.vocab_dict:
             for pos, ch in enumerate(w):
                 pos_word_chr[pos, self.vocab_dict[w], self.characters.index(ch)] = 1
         return(pos_word_chr)
 
 class OneVirtualReader:
+    """ A virtual reader with a vocabulary and some visual properties.
+
+    Attributes:
+        vocabulary: A Vocabulary class storing language knowledge.
+        sigma_scale: A float that controls shape of visual acuity function; small number indicates narrow acuity function.
+        Lambda_scale: A float that controls overall visual input quality; small number indicates poor visual input.
+        lpos_range: A list of numbers indicating landing position range.
```

Comment:
an example would help make this one clearer! Are there any constraints on it? Should it just be a min and max?

Reply:
changed variable name to `fix_loc_list` to avoid confusion. it's a list of fixation location, not min and max.

Reply:
i think i'm still confused... and I left a comment in the code. When it's given a value like [1, 7] isn't that supposed to be interpreted as a range?

Reply:
sorry, nope, [1, 7] means position 1 and 7, which is not supposed to be interpreted as min and max. (should be [1,2,3,4,5,6,7] if they'd like all letter positions). Maybe I should use other numbers in the usage example (e.g. [1,4,7]) to avoid such confusion?

Reply:
ok, got it. Why would anyone want to create a reader that is only allowed to fixate positions 1 and 7? This seems like an unintuitive use case

Reply:
I guess the follow-up is that if most of the time, users want to allow a reader to fixate every position in the word, storing the possible positions in a list doesn't seem like the right data structure. Or am I still misunderstanding something?

Reply:
I think my motivation is that using `range(1, wlen+1)` to get a list of numbers in a range is simple enough, and we can reduce unnecessary computation if the user just want a few landing positions.

Reply:
ok that makes sense! I'd just make this trade-off explicit and clear in the documentation (and example scripts). This could include adding a comment like `# limit possible fixation locations to positions 1 and 7 to reduce computation` in the example scripts.

Reply:
probably also worth pointing out somewhere that the user may want to use negative possible locations if their use case can involve fixating off of the word (e.g., due to motor error)